airflow/api_fastapi/core_api/routes/public/variables.py

```diff
@@ -160,12 +160,23 @@ def patch_variable(
 @variables_router.post(
     "",
     status_code=status.HTTP_201_CREATED,
+    responses=create_openapi_http_exception_doc(
+        [status.HTTP_409_CONFLICT, status.HTTP_422_UNPROCESSABLE_ENTITY]
```

Comment:
422 is automatically added by fastapi on all endpoints.

We don’t need to add it manually.

Reply:
Oh, okay